specificatie/openapi.yaml

```diff
@@ -648,9 +770,65 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    Cirkel:
+      type: object
+      description: 'Om met een cirkel te zoeken moet je een punt en een straal (in meters) opgeven.'
+      properties:
+        punt:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+        radius:
+          type: number
+          default: 100
+          minimum: 1
+          maximum: 280
+
+    BoundingBox:
+      type: object
+      description: 'Om met een bounding box te zoeken moet je de coördinaten van links boven en rechts onder opgeven.'
+      properties:
+        coordinates:
+          type: array
+          minItems: 2
+          maxItems: 2
+          items:
+          type: array
+            minItems: 2
+            maxItems: 2
+          items:
+          type: number
+              example:
+              - 196733.510
+              - 439931.890
+
+    IntegerFilter:
```

Comment:
.NET code generatie kan niet omgaan met de anyOf constructie. In geval van oppervlakte blijft alleen de int type over. De oppervlakte parameter ziet er dus als volgt uit: int? oppervlakte. Hier moet wat anders voor worden bedacht

Reply:
Aangepast, graaf even controleren of het met de nieuwe versie wel gaat.

Reply:
Er wordt correcte code gegenereerd, maar SwaggerUI en Postman kunnen niet met deze nieuwe definitie omgaan. Swagger UI toont alleen een text veld. Geen idee hoe je hier min, max of not kan aangeven en in postman komt de oppervlakte property helemaal niet voor in de Params lijst